### PR TITLE
Add OAuth

### PR DESCRIPTION
**Pull request by miyamoen**
done ja

### CODE REVIEW

**Comment by miyamoen** [comment]
done en

File: src/content/auth.md [hunk 1]
```diff
@@ -66,3 +66,40 @@ Next, request with "Authorization: Token 5f080f9c-3023-4a6a-89fb-de983f768c4d" h
 curl -XGET -H 'Content-Type: application/edn' -H 'Authorization: Token 5f080f9c-3023-4a6a-89fb-de983f768c4d' http://localhost:45102/default/jobs
 {:results [...]}
 ```
+
+## OAuth2.0 Authentication
+
+OAuth2.0 is available for authentication when you use JobStreamer.
```
Comment (223kazuki) [inline src/content/auth.md]: You can use OAuth2.0 as authentication in JobStreamer.

File: src/content/auth.md [hunk 2]
```diff
@@ -66,3 +66,40 @@ Next, request with "Authorization: Token 5f080f9c-3023-4a6a-89fb-de983f768c4d" h
...
+If you use OAuth2.0, you login as a "guest" user with operator role.
```
Comment (223kazuki) [inline src/content/auth.md]: If you login to JobStreamer by using OAuth2.0, You will get the "operator" role.

File: src/content/auth.md [hunk 3]
```diff
@@ -66,3 +66,40 @@ Next, request with "Authorization: Token 5f080f9c-3023-4a6a-89fb-de983f768c4d" h
...
+
+### How to use
+
+You locate configuration file on class path.
```
Comment (223kazuki) [inline src/content/auth.md]: To Use OAuth2.0, You have to locate a configuration file on your class path.

File: src/content/auth.md [hunk 4]
```diff
@@ -66,3 +66,40 @@ Next, request with "Authorization: Token 5f080f9c-3023-4a6a-89fb-de983f768c4d" h
...
+                          "github" {:name "Github"
+                                    :class-name "github"
+                                    :domain "https://github.com"
+                                    :client-id "xxxx"
+                                    :client-secret "xxxx"
+                                    :auth-endpoint "login/oauth/authorize"
+                                    :token-endpoint "login/oauth/access_token"}}}}
+```
+
+You must configure in OAuth2.0 authorization server.
```
Comment (223kazuki) [inline src/content/auth.md]: And you have to configure the OAuth2.0 authorization server.

File: src/content/auth.md [hunk 5]
```diff
@@ -66,3 +66,40 @@ Next, request with "Authorization: Token 5f080f9c-3023-4a6a-89fb-de983f768c4d" h
...
+
+Checked available OAuth2.0 authorization servers.
```
Comment (223kazuki) [inline src/content/auth.md]: We confirmed that it works with the following OAuth2.0 services.

**Review by 223kazuki — COMMENTED** [review]
Fixed.

**Pull request merged**